yarn.lock

```diff
@@ -1239,6 +1239,14 @@ react-cookie-consent@^8.0.1:
   dependencies:
```

Comment:
will we use yarn i future or just npm since build is connected to npm also?
@tomesfilip

Reply:
we can use yarn. It's just package manager tool just like npm, it will run the build command that executes another command. These are defined in package.json under "scripts"

```
"scripts": {
    "dev": "vite",
    "build": "tsc && vite build",
    "preview": "vite preview"
  },
```